src/components/OptionsForm.tsx

```diff
@@ -39,44 +50,71 @@ export function OptionsForm({ options }: OptionsFormProps) {
 			</div>
 
 			<div className={styles.secondaryOptionsArea}>
-				<h2 className={styles.h2}>optionally, filter by:</h2>
-				<div className={styles.secondaryOptions}>
-					<div className={styles.secondaryOptionArea}>
-						<label className={styles.labelSecondary} htmlFor="since">
-							Updated Since
-						</label>
-						<input
-							className={styles.inputSecondary}
-							defaultValue={options.since && options.since.toString()}
-							id="since"
-							name="since"
-							type="date"
-						></input>
-					</div>
+				<button
```

Comment:
[Bug] A `<div>` containing a `<button>` _visually_ looks fine, but _semantically_ isn't quite what's happening. This causes an accessibility bug: screenreader users won't receive any indication of what changes when the button is pressed.

I think what you want here is a `<details>` + `<summary>`. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details - this is the browser native way of doing it. I tested it out locally and although the styles were broken, the functionality worked. Could you please switch to that? (unless I'm wrong and there's a better accessibility fix?)

i.e. `<details className={...} open={isSecondaryOptionsVisible}>`

Reply:
Great catch! After doing some research, it looks like you can also use aria attributes to semantically connect the `<div>` and `<button>` elements but I think using the `<details>` element is a more clean and straight-forward approach.